### ISSUE TITLE
Algorithms and code snippets in C.

### ISSUE THREAD

**Issue by amanmehara** [amanmehara/programming#56]

**Comment by gbasupi** [comment]
> _No description provided._

**Comment by marcotaranta** [comment]
Made a commit for doubly linked lists in C

**Comment by abumaz** [comment]
Hey, Can I be assigned to work on Banker's Algorithm problem for this issue? I noticed that it's not there here and I would like to contribute to Hacktoberfest too.

**Comment by amanmehara** [comment]
@abumaz definitely.
Create a PR with the algorithm implementation.

Also have a look at https://github.com/amanmehara/programming/blob/master/CONTRIBUTING.md and make changes accordingly.
I'll approve and merge afterwards! 😃

**Comment by Dhruv-VINT** [comment]
Can we add algorithms for C++ too in this repo?

**Comment by amanmehara** [comment]
@Dhruv-VINT yes, you can add, just create a PR having algorithms implemented in C++. 😃
